src/main/kotlin/demo/services/RecruitmentProcessService.kt

```diff
@@ -31,4 +32,8 @@ class RecruitmentProcessService(@Autowired private val recruitmentProcessReposit
 
     fun deleteProcess(id: Int) = recruitmentProcessRepository.deleteById(id)
 
+    fun getProcessFromStage(recruitmentProcessStage: RecruitmentStage): RecruitmentProcess {
+        return recruitmentProcessRepository.findAll().find { it.stages.contains(recruitmentProcessStage) }!!
```

Comment:
Tylko się upewniam: to na potrzeby łatania wersji prototypowej, nie ma generalnie takiej zależności że możemy mieć tylko jeden proces na danym etapie? xD

Reply:
Yyy każdy stage jest unikalny. I proces rekrutacyjny nie jest przyporządkowany że jest na jakimś etapie, tylko ma zdefiniowane jakie etapy w tym procesie są możliwe. To aplikacja jest zdefiniowana że jest na etapie, rozmowy albo dopiero zaaplikowano.

Reply:
Hope so XD

Reply:
okej, jak id tez bierzemy pod uwage to spk xD

Reply:
Bierze uwagę bo obstawiam że contains działa na equals, które są domyślnie implementowane w data class